crates/types/Cargo.toml

```diff
@@ -20,6 +20,7 @@ version = { workspace = true }
 anyhow = { workspace = true }
 derive_more = { version = "0.99" }
 fuel-vm-private = { workspace = true }
+getrandom = { version = "0.2", features = ["js"] }
```

Comment:
See: https://docs.rs/getrandom/latest/getrandom/#webassembly-support

`getrandom` is a transitive dependency of `fuel-crypto`.